modules/sno-du-prerequisites.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
```

Comment:
I don't see this module (or the assembly listed here) in the preview. But also I just got hit with the CSS outage so I am finding troubleshooting/checking this to be quite difficult :upside_down_face: Just calling it out as an FYI.

Reply:
This module was commented out of the assembly but not yet removed from the repo. I'll remove it.